DESCRIPTION

```diff
@@ -60,6 +60,8 @@ Suggests:
     MultiAssayExperiment,
     R6,
     rmarkdown (>= 2.19),
+    rvest,
```

Comment:
`{glue}` needs to be added here, alternatively, we could use `{stringr}` and use `stringr::str_glue()` instead of `glue::glue` _(as you mentioned that `{stringr}` could be useful)_

They also need to be added below to the "Config/Need/verdepcheck"

`tidyverse/rvest, rstudio/shinytest2, tidyverse/glue`

```suggestion
    rvest,
    glue,
```

Reply:
We can also use `sprintf` which you probably noticed I prefer to use. This is why:

```r
library(glue)
val1 <- "elo1"
val2 <- "elo2"
val3 <- "elo3"

elo <- microbenchmark::microbenchmark(
  sprintf = sprintf("%s %s %s", val1, val2, val3),
  paste = paste(val1, val2, val3),
  glue = glue("{val1} {val2} {val3}")
)

ggplot2::autoplot(elo)
``` 

![image](https://github.com/insightsengineering/teal/assets/6959016/3c0a74e6-7029-4276-a028-913b369500bd)

Reply:
wow

Reply:
`paste` on my laptop is almost on par with `sprintf`, but I'm switching teams to `sprintf` anyway :grin:

``` r
elo <- microbenchmark::microbenchmark(
  sprintf = sprintf("%s %s %s", val1, val2, val3), 
  paste = paste(val1, val2, val3), 
  glue = glue("{val1} {val2} {val3}"),
  strc_c = stringr::str_c(val1, val2, val3, sep = " "),
  glue2 = glue(val1, val2, val3, .sep = " "),
  times = 100L
) 
```

![image](https://github.com/insightsengineering/teal/assets/211358/033d7152-3f3e-4ac3-93b7-ef65a3ab1e25)

Reply:
Now we are using `sprintf` throughout 💪🏽